src/CatGFX.h

```diff
@@ -52,6 +51,11 @@ class CatPrinter: public Adafruit_GFX, public BLEAdvertisedDeviceCallbacks {
     BLEScan *bleScan = BLEDevice::getScan();
     BLEClient *bleClient = BLEDevice::createClient();;
 
+	//Indicates the end of the array with an empty string instead of NULL.
+	const static uint8_t NAME_ARRAY_SIZE = 6;
+	const static uint8_t NAME_STRING_SIZE = 8;
+	char printer_names[NAME_ARRAY_SIZE][NAME_STRING_SIZE];
```

Comment:
Minor thing, but doesn't fit naming I used for the other variabled. printerNames would be a better fit.